.circleci/config.yml

```diff
@@ -35,22 +35,22 @@ workflows:
           <<: *build_always
       - build_dashboard:
           <<: *build_always
-      - GKE_1_9_MASTER:
-          <<: *build_on_master
+      - GKE_1_9:
```

Comment:
any reason you chose to remove the MASTER suffix? I like that it is more clear

Reply:
I agree

Reply:
I removed the `MASTER` suffix because before this was only executed on master but now it's run in any branch. I thought of using `LATEST` but it's quite similar to `LATEST_RELEASE`. I will try with different suffixes if you think this is not very clear.

Reply:
> I removed the MASTER suffix because before this was only executed on master but now it's run in any branch.

Why is that? Why have we changed that behavior?

> I thought of using LATEST but it's quite similar to LATEST_RELEASE. I will try with different suffixes if you think this is not very clear.

The reason of using `LATEST_RELEASE` is because in my opinion, LATEST means latest version of the code, while just RELEASE can mean testing the new release.

I see that you used RELEASED (in past tense) I think it's fine but I still prefer the more explicit way of `LATEST_RELEASE`